src/backend/joanie/tests/core/test_utils_payment_schedule.py

```diff
@@ -103,6 +103,23 @@ def test_utils_payment_schedule_withdrawal_higher_than_course_start_date(self):
             date(2024, 1, 1),
         )
 
+    def test_utils_payment_schedule_withdrawal_higher_than_course_start_date_even_weekend(
+        self,
+    ):
+        """
+        If the withdrawal date is after the course start date, the return date should be the
+        signed contract date
+        """
+        signed_contract_date = date(2023, 12, 14)
```

Comment:
Can you also test with a signed date corresponding to a saturday or sunday ? This is the bug you are fixing

Reply:
Actually, this is the purpose of this test. A comment is missing to explain that if the signed_contract_date is the 14 dec. 2023 and the withdraw period is 16 days so the withdrawal limit date will be a saturday (Sat. 30 dec. 2023).